generators/deployment/templates/pipeline_master.yml

```diff
@@ -14,6 +14,18 @@ stages:
   - name: CHART_NAME
     value: ${CHART_NAME}
     type: text
+  - name: KUBECTL_VERSION
+    value: 1.10.11
```

Comment:
What versions are predefined here?   Are these the latest, are they the default for devops, or just arbitrary version numbers?  In my issue description, I just picked arbitrary version numbers to demonstrate the point.

Reply:
Also, just to be clear, are `${KUBECTL_VERSION}` and `${HELM_VERSION}` evaluated in the urls at the time of download, or are those literals in the url request?